.github/workflows/Main.yml

```diff
@@ -95,7 +95,7 @@ jobs:
       CXX: g++-10
       GEN: ninja
       BUILD_JEMALLOC: 1
-      CORE_EXTENSIONS: "icu;parquet;tpch;tpcds;fts;json"
+      CORE_EXTENSIONS: "icu;parquet;tpch;tpcds;json"
```

Comment:
They don't work because you have a DONT_LINK in [.github/config/out_of_tree_extensions.cmake](https://github.com/duckdb/duckdb/pull/14872/files#diff-0024509856f4c4e05febb4181280cf8f8d10e02b426c911fd630b3f11fcb0c54).
I think I would remove that, and see if stuff works, since there might be legitimate use cases for statically link in FTS.

Reply:
Also in a follow up, this is not critical. But I would change it back so that same set of tests is run.

Reply:
Is `DONT_LINK` really the reason for this error?
```
Failed to load database duckdb_unittest_tempdir/10483/_home_runner_work_duckdb_duckdb_build_reldebug__deps_fts_extension_fc-src_test_sql_fts_issue_13866_test.db 
================================================================================
Error message: IO Error: Cannot open file "duckdb_unittest_tempdir/10483/_home_runner_work_duckdb_duckdb_build_reldebug__deps_fts_extension_fc-src_test_sql_fts_issue_13866_test.db": No such file or directory
```
It seems like the db test path of an out-of-tree extension is handled differently, which trips up Force Storage/Force Restart because they create a db by default. I think that the `duckdb_unittest_tempdir/10483` directory is not created in the out-of-tree extension test path, only in the main repo test path.